praw/models/reddit/submission.py

```diff
@@ -341,6 +341,43 @@ def unspoiler(self):
         """
         self.thing._reddit.post(API_PATH["unspoiler"], data={"id": self.thing.fullname})
 
+    def update_crowd_control_level(self, level: int):
+        """Change the Crowd Control level of the submission.
+
+        :param level: An integer between 0 and 3.
+
+        **Level Descriptions**
+
+        ===== ======== ================================================================
+        Level Name     Description
+        ===== ======== ================================================================
+        0     Off      Crowd Control will not action any of the submission's commments.
+        1     Lenient  Comments from users who have negative karma in the subreddit are
+                       automatically collapsed.
+        2     Moderate Comments from new users and users with negative karma in the
+                       subreddit are automatically collapsed.
+        3     Strict   Comments from users who haven’t joined the subreddit, new users,
+                       and users with negative karma in the subreddit are automatically
+                       collapsed.
+        ===== ======== ================================================================
+
+        Example usage:
+
+        .. code-block:: python
+
+            submission = reddit.submission(id="745ryj")
+            submission.mod.update_crowd_control_level(2)
+
```

Comment:
Could we add a note that if integers outside 0-3 are passed reddit will use the closest valid integer.

Reply:
I'm hesitant to do this because it's undocumented behavior that doesn't really have a use. It's just something I came across during testing.

Reply:
Good point. I think it would be fine as long as we added a test for it and say in the note the behavior is what currently happens and could throw an exception in the future. I'm fine with it either way, though its current behavior is unexpected to me as I'd expect an error thrown when an invalid level is passed.

@bboe what do you think?

Reply:
I don't think PRAW should do anything specifically here. We don't check that usernames or subreddit names are allowable names and instead rely upon Reddit to perform that validation. I like having that level of simplicity in our library. Yes, that means something could break for someone who is using a value outside of the documented range, but if they're using a value outside of the documented range then the resulting behavior could reasonably be unexpected.

Reply:
I agree that we shouldn't do client side validation. My suggestion was for documenting Reddit's behavior to use the closest valid integer in this instance.

Reply:
I don't think we should mention that behavior lest it change.

Reply:
Sounds good!